### ISSUE TITLE
Adapt the config.yml instead of initconfig.php

### ISSUE THREAD

**Issue by netzarbeiter** [contao/core-bundle#1437]
If I want to add some specific CSS for the backend, the way I know is:

Writing the following lines in the `system/config/initconfig.php`

```php
if(TL_MODE == 'BE')
{
    $GLOBALS['TL_CSS'][]        = 'layout/css/backend.css';
}
```

Would it not be nice if we could reference to the file in the config.yml?

e.g.
```yml
contao:
    additional_backend_css:
        - layout/css/backend.css
```

**Comment by Toflar** [comment]
I think this is a valid use case. It's pretty complicated right now for non-developers to customize the back end layout a little. I think one css file would be enough to cover 90% of all use cases and more complex ones should use hooks.

**Comment by bezin** [comment]
I like this. We sometimes develop projects with Kirby, and they solve it exactly that way: https://getkirby.com/docs/developer-guide/panel/css. Really handy for minor changes.

**Comment by aschempp** [comment]
I could generally agree to something like that, but I think it should be something like this:

```yml
contao:
    backend:
        additional_css: 'layout/css/backend.css'
        additional_js: 'layout/js/backend.js'
```

I don't think there is a need for multiple files, and we should allow both CSS and JS.

**Comment by m-vo** [comment]
(btw.: I'd even prefer to have something like this when developing bundles. That looks a lot cleaner than using the global arrays and would suffice in many cases.)

**Comment by Toflar** [comment]
It's actually an easy pick. Anyone welcome to work on that.

**Comment by aschempp** [comment]
> (btw.: I'd even prefer to have something like this when developing bundles. That looks a lot cleaner than using the global arrays + context and would suffice in many cases.)

That's a completely different scope. The bundle configuration is for application specific adjustments, never for bundles!

**Comment by leofeyer** [comment]
Another way would be to create a custom `be_main.html5` template:

```php
<?php $this->extend('be_main'); ?>

<?php $this->block('head'); ?>
  <?php $this->parent(); ?>
  <link rel="stylesheet" href="layout/css/backend.css">
<?php $this->endblock(); ?>
```

Should be 100% update safe. Do we still need a third way to accomplish this?

**Comment by netzarbeiter** [comment]
Thank you, but your solution is not working in Contao 4.4.30.
If it is working in Contao 4.7.0 feel free to close the ticket.

**Comment by leofeyer** [comment]
It works as of Contao 4.5.

@contao/developers What do you think?

**Comment by bezin** [comment]
I think I would still prefer the way @netzarbeiter sketched out in its first post. In the end we migrate away from config.php, initconfig.php and dcaconfig.php in the long run and hence I would like the ability to just define it in the newer config files like config.yml

**Comment by rabauss** [comment]
I will try to solve this in parallel with https://github.com/contao/contao/pull/1999

**Comment by rabauss** [comment]
Can be closed with contao/contao#1999
